CSharp/BatchMetrics/BatchMetricsUsageSample/packages.config

```diff
@@ -1,13 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Azure.Batch" version="5.1.2" targetFramework="net452" />
+  <package id="Azure.Batch" version="6.0.0" targetFramework="net452" />
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net452" />
-  <package id="Microsoft.Data.Edm" version="5.6.4" targetFramework="net452" />
-  <package id="Microsoft.Data.OData" version="5.6.4" targetFramework="net452" />
-  <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.1" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.1" targetFramework="net452" />
+  <package id="Microsoft.Data.Edm" version="5.7.0" targetFramework="net452" />
+  <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
+  <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.5" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.5" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net452" />
-  <package id="System.Spatial" version="5.6.4" targetFramework="net452" />
-  <package id="WindowsAzure.Storage" version="6.2.0" targetFramework="net452" />
+  <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
+  <package id="WindowsAzure.Storage" version="8.0.1" targetFramework="net452" />
```

Comment:
Again, not sure we need Storage here, or is it that the common project brings it in?